src/components-v2/dataDisplay/TextBox/index.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import React from 'react'
+import styled from 'styled-components'
+
+type Props = {
+  children: React.Node,
+}
+
+const Box = styled.div`
+  padding: 10px;
+  word-wrap: break-word;
+  border: solid 2px #e8e7e6;
```

Comment:
this color is already a variable

```
safe-react/src/theme/variables.js
```

would it be possible to use it?

Reply:
sure

Reply:
done!